sw/host/opentitanlib/src/otp/alert_handler_regs.rs

```diff
@@ -0,0 +1,1215 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+/* automatically generated by rust-bindgen 0.60.1 */
```

Comment:
awesome! Can you regenerate this file because 621b69ea0c35c5d30373755f65fc4108fbceb0dc adds new alerts and I'm not sure if they are included here?

Also, can you
- [ ] Add the script that is used to generate this file so that we can regenerate it easily if needed, and
- [ ] Create an issue (high priority) to add a CI check for this file? I think we already have a similar check in the CI? @timothytrippel, right?

Thanks!

Reply:
yeah thats right: https://cs.opensource.google/opentitan/opentitan/+/master:ci/scripts/check-generated.sh